util/config/config.go

```diff
@@ -74,6 +74,7 @@ const (
 
 var (
 	Debug            = false
+	Pruning          = false
```

Comment:
Suggest change `Pruning` to `StatePruning`

Reply:
ok, I will do it.